altair/utils/html.py

```diff
@@ -218,6 +251,22 @@ def spec_to_html(
     if mode == "vega-lite" and vegalite_version is None:
         raise ValueError("must specify vega-lite version for mode='vega-lite'")
 
+    render_kwargs = dict()
+    if template == "inline":
+        try:
+            from altair_viewer import get_bundled_script
+        except ImportError:
+            raise ImportError(
+                "The altair_viewer package is required to convert to HTML with inline=True"
+            )
```

Comment:
I like this approach versus adding `altair_viewer` as a required dependency for altair